### PR TITLE
New package: peaclock-0.1.3

### PR DESCRIPTION
**Pull request by Crestwave**

### CODE REVIEW

File: srcpkgs/peaclock/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'peaclock'
+pkgname=peaclock
+version=0.1.3
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=debug"
```
Comment (Johnnynator) [inline srcpkgs/peaclock/template]: Why do you set the build type to debug? Release is normally what we want and is also the default of the cmake build_style, so remove this line.

File: srcpkgs/peaclock/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+hostmakedepends="cmake"
+short_desc="A colourful binary clock for the terminal"
```
Comment (Johnnynator) [inline srcpkgs/peaclock/template]: Just "Colourful binary clock for the terminal" should be enough

File: srcpkgs/peaclock/template (hunk 1 above)
Comment (Crestwave) [reply srcpkgs/peaclock/template]: I thought that binaries should include debug info?

Comment (Crestwave) [reply srcpkgs/peaclock/template]: I checked when I added it, and Release mode did strip the binaries.

Comment (Crestwave) [reply srcpkgs/peaclock/template]: Ah, I forgot about that build type. I've removed it now.

Comment (Johnnynator) [reply srcpkgs/peaclock/template]: #7004

**Comment by Crestwave** [comment]
@ndowens Whoops; sorry! It wasn't there when I first checked and I forgot to check again before submitting the template.

**Pull request closed without merging**